variables.tf

```diff
@@ -57,3 +57,9 @@ variable "lacework_integration_name" {
   default     = "TF ECR IAM Role"
   description = "The name of the external ECR integration"
 }
+
```

Comment:
nit: Should we be using the same description found [here](https://registry.terraform.io/providers/lacework/lacework/latest/docs/resources/integration_ecr#non_os_package_support).

Reply:
It is the same, but the default is automatically generated, look at the registry.